examples/nixnet/can-signal-single-point-output.py

```diff
@@ -0,0 +1,125 @@
+Client" wiki page:
+  https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Refer to the NI XNET gRPC Wiki for the latest C Function Reference:
+  https://github.com/ni/grpc-device/wiki/NI-XNET-C-Function-Reference
+
+ Running from command line:
+Server machine's IP address and port number can be passed as separate command line
+arguments.
+  > python can-signal-single-point-output.py <server_address> <port_number>
+If they are not passed in as command line arguments, then by default the server address will be
+"localhost:31763"
+"""
+
+import sys
+
+import getch
+import grpc
+import nixnet_pb2 as nixnet_types
+import nixnet_pb2_grpc as grpc_nixnet
+
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    SERVER_ADDRESS = sys.argv[1]
+if len(sys.argv) >= 3:
+    SERVER_PORT = sys.argv[2]
+
+# Parameters
+INTERFACE = "CAN1"
+DATABASE = "NIXNET_example"
+CLUSTER = "CAN_Cluster"
+SIGNAL_LIST = "CANEventSignal1,CANEventSignal2"
+NUM_SIGNALS = 2
+SUCCESS = 0
+
+
+def check_for_error(status):
+    """Raise an exception if the status indicates an error."""
+    if status != 0:
+        error_message_response = client.StatusToString(
+            nixnet_types.StatusToStringRequest(status_id=status)
+        )
+        raise Exception(error_message_response.status_description)
+
+
+i = 0
+value_buffer = [0.0] * NUM_SIGNALS
+
+# Create the communication channel for the remote host and create connections to the NI-XNET and
+# session services.
+channel = grpc.insecure_channel(f"{SERVER_ADDRESS}:{SERVER_PORT}")
+client = grpc_nixnet.NiXnetStub(channel)
+
+# Display parameters that will be used for the example.
+print("Interface: " + INTERFACE, "Database: " + DATABASE, "Signal List: " + SIGNAL_LIST, sep="\n")
+
+try:
+    # Create an XNET session in SignalOutSinglePoint mode
+    create_session_response = client.CreateSession(
+        nixnet_types.CreateSessionRequest(
+            database_name=DATABASE,
+            cluster_name=CLUSTER,
+            list=SIGNAL_LIST,
+            interface=INTERFACE,
+            mode=nixnet_types.CREATE_SESSION_MODE_MODE_SIGNAL_OUT_SINGLE_POINT,
+        )
+    )
+    check_for_error(create_session_response.status)
+
+    session_reference = create_session_response.session_ref
+
+    print("Session Created Successfully.\n")
+    print("\nPress any key to transmit new signal values or q to quit\n")
+
+    while not (getch.getch()).decode("UTF-8") == "q":
+        value_buffer[0] = float(i)
+        value_buffer[1] = float(i * 10)
+
+        # Update the signal data
+        write_signal_response = client.WriteSignalSinglePoint(
+            nixnet_types.WriteSignalSinglePointRequest(
+                session_ref=session_reference, value_buffer=value_buffer
+            )
+        )
+        check_for_error(write_signal_response.status)
+
+        print("Signals sent:")
```

Comment:
Check for understanding: you say in the PR that the output is the same as the C API. But this is only printing data from the write buffer. So all that validates (for now) is that write didn't return an error?

Reply:
No, the function `check_for_error` raises a exception whenever there is a error and the print statements will be executed only if the status doesn't indicate a error.

Reply:
That's what I was thinking, sorry if my message was not clear.

That sounds fine.

Reply:
@astarche  I have reset your vote. Could you please review and approve the changes once again?